docs/Using_Benchmark.md

```diff
@@ -0,0 +1,285 @@
+# Using Benchmark
+
+If we want to use Benchmark with a particular file, 
+say `BenchmarkEverything.cpp`, we can build Benchmark and use the following 
+commands in the terminal.
+
+## In Windows
+```bash
+> cl /EHsc /MD BenchmarkEverything.cpp /I<BENCHMARK_SOURCE_DIR>\include /link shlwapi.lib <BENCHMARK_BUILD_DIR>\src\Release\benchmark.lib
+```
+where, `<BENCHMARK_SOURCE_DIR>` is the directory where Benchmark was cloned into
+and `<BENCHMARK_BUILD_DIR>` the directory where Benchmark was built. Here, we 
+had built Benchmark in the Release configuration and hence we used 
+`<BENCHMARK_BUILD_DIR>\src\Release\benchmark.lib`. We also used the `cl` 
+compiler which is provided with Visual Studio and also linked the `shlwapi` 
+library.
+
+## In Linux
```

Comment:
This is super flame-wary and nitpicky, but i think linux should be explained first.

Reply:
its the same